wp-content/themes/core/assets/css/src/theme/typography/_utilities.pcss

```diff
@@ -1,66 +1,125 @@
 /* -----------------------------------------------------------------------------
  *
- * Utility Classes: Typography Color Themes
+ * Utility Classes: Typography
  *
  * ----------------------------------------------------------------------------- */
 
 /* -------------------------------------------------------------------------
- *
- * Theme: Light
- *
- * Notes:
- * + This can be used anywhere or can be paired up with .t-content
- * + Also see the other base partials for use of this class as a parent
- *   selector (media, typography, etc.)
- *
- * Example:
- * <div class="t-content--light">
- *   ...
- *   <div class="t-content">
- *     ...
- *   </div>
- * </div>
- *
- * Example:
- * <div class="t-content t-content--light">
- *   ...
- * </div>
- *
+ * Display X-Large
```

Comment:
Nothing to change, noting for future to follow up on this, wondering if it's valuable or not to have all of these as helper classes that then output a bunch of CSS, especially if we are going to 99% of the time us our sink helpers as these won't get us much mileage without responsive handling and as of now we both agree we don't want to go down that wormhole.

Reply:
Hmm... design tends to use these "base" typographic styles many time throughout the designs where there is no responsive treatment. I've found this system super helpful on the last few projects. Happy to reevaluate as we proceed, but for now, I'd like to keep them available.

Reply:
👍